src/PharoLauncher-CLI-Tests/PhLImageKillCliCommandTest.class.st

```diff
@@ -0,0 +1,18 @@
+"
+A PhLImageKillCliCommandTest is a test class for testing the behavior of PhLImageKillCliCommand
+"
+Class {
+	#name : #PhLImageKillCliCommandTest,
+	#superclass : #PhLImageCliCommandTest,
+	#category : #'PharoLauncher-CLI-Tests'
+}
+
+{ #category : #tests }
+PhLImageKillCliCommandTest >> testKillImageCmdStringFrom [
```

Comment:
Is this test really useful? It is so dependant on the implementation.

Reply:
I agree, probably it is too strict for checking exact content of this. Purpose of this test was to test if cascading of grep with image name parameter works correctly or not (and method is not failing). I will rewrite test case.